docs/content/Infrastructure.md

```diff
@@ -145,8 +145,8 @@ is version for future release.
 
 [Bintray project](https://bintray.com/avito-tech/maven/avito-android), mirroring to jcenter
 
-1. Make sure integration tests passed via `CI integration tests against avito`
-1. Make sure new project version specified in develop head
+1. Checkout branch with name equally to `projectVersion`
```

Comment:
why is that? just a convention? nothing seems rely on it for now

Reply:
@sboishtyan , ?